### PR TITLE
Experimental support for Structured Discussions

### PR DESCRIPTION
**Pull request by lens0021**
This will add a hook for [Structured Discussions], but with limitations:

- Only works with actions from Javascript UI; non Javascript actions will be ignored. This happens in browsers do not support Javascript.
- The title of topics is only displayed as UUID (for instance, `Topic:V2x4eocpudm5wtz4`) rather than a human readable ones.

In addition, my source editor removes some whitespaces automatically. But it affects nothing, don't worry about it.

[Structured Discussions]: https://www.mediawiki.org/wiki/Structured_Discussions

### CODE REVIEW

File: DiscordNotificationsCore.php [hunk 1]
```diff
@@ -470,6 +562,11 @@ private static function send_http_request($url, $postData) {
 
     private static function getMessage($key) {
 		return wfMessage( $key)->inContentLanguage()->text();
-    }
+		}
+
+	private static function flowUUIDToTitleText($UUID) {
+		// TODO
```
Comment (kulttuuri) [inline DiscordNotificationsCore.php]: Implemented?

Comment (lens0021) [reply DiscordNotificationsCore.php]: It is the unimplemented part I mentioned as "The title of topics is only displayed as UUID" in the descrition. I should find the way for this. :)

**Comment by kulttuuri** [comment]
I've pushed the commits manually to master to prevent any whitespace / formatting changes to files. Can you try it and let me know that it works? :)

**Comment by lens0021** [comment]
It works for me, Thank you!

**Pull request closed without merging**